routes/users.js

```diff
@@ -0,0 +1,59 @@
+const express = require('express');
+const knex = require('../database');
+const router = express.Router();
+
+router.route('/:user_id')
+  .get((req, res) => {
+    knex.raw('SELECT * FROM users WHERE id = ?', [req.params.user_id])
+    .then((result) => {
+      if (result.rowCount <= 0){
+        throw { message : 'User not found' };
+      }
+      res.send(result);
+    })
+    .catch((err) => {
+      res.send(err);
```

Comment:
You should be sending back a 500 response at this point. Please reserve this space for any miscellaneous errors from the server (i.e. can't reach the server, database table doesn't exists). Any errors you can handle in your promise chain, you should handle them up above rather than in here.